packages/app/src/components/DBRowSidePanel.tsx

```diff
@@ -216,6 +218,18 @@ export default function DBRowSidePanel({
     enabled: rowId != null,
   });
 
+  const hasK8sContext = useMemo(() => {
+    if (!source?.resourceAttributesExpression || !normalizedRow) {
+      return false;
+    }
+    return (
+      normalizedRow[source.resourceAttributesExpression]['k8s.pod.uid'] !=
```

Comment:
We can't count on the return value being the same as the expression the user initially inputted, which will lead to bugs. If we want to extract a specific value in the frontend, we should alias it to a specific string and then use the aliased value.